build/Dockerfile-setup

```diff
@@ -5,7 +5,7 @@ RUN apk update; apk add postgresql-client mysql-client curl python tar
 
 RUN  mkdir /opt; \
      cd /opt ; \
-     curl http://archive.apache.org/dist/cassandra/3.3/apache-cassandra-3.3-bin.tar.gz | tar zx
+     curl https://www.apache.org/dist/cassandra/3.3/apache-cassandra-3.3-bin.tar.gz | tar zx
```

Comment:
Apache archive is in maintenance. Revert this part after Monday. 

We are in the process of relocating this service. To save on the immense bandwidth that this service outputs, we have put it in maintenance mode, disabling all downloads for the next few days. We expect the maintenance to be complete no later than the morning of Monday the 11th of April, 2016.